src/autopas/selectors/tuningStrategy/GaussianModel/AcquisitionFunction.h

```diff
@@ -0,0 +1,60 @@
+/**
+ * @file AcquisitionFunction.h
+ * @author Jan Nguyen
+ * @date 13.04.20
+ */
+
+#pragma once
+
+#include <Eigen/Core>
+#include <utility>
+
+#include "autopas/options/AcquisitionFunctionOption.h"
+#include "autopas/utils/ExceptionHandler.h"
+#include "autopas/utils/Math.h"
+
+namespace autopas {
+
+/**
+ * Functions used for acquisition prediction used in Gaussian Process models
+ */
+class AcquisitionFunction {
```

Comment:
This does not need to be a `class`. Make it a `namespace` (as for example in `OptimumSelector.h`.